content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Prerequisites_and_Preparations.adoc

```diff
@@ -1,36 +1,40 @@
 == Prerequisites and Preparations
 
-The audience of this document is an experienced systems administrator with a good understanding of Microsoft Windows Server management. In addition, it is helpful to be familiar with the terminology, software, and tools specific to YubiHSM 2. As a primer for these, refer to the Terminology Used chapter.
+The audience of this document is an experienced systems administrator with a good understanding of Microsoft Windows Server management. In addition, it is helpful to be familiar with the terminology, software, and tools specific to YubiHSM 2. As a primer for these, refer to the Terminology table in link:Getting_Help.adoc[Getting Help].
 
 In order to follow the steps provided in this guide, the following prerequisites must be met:
 
 * Access to Microsoft Windows Server 2012 SP2 or higher, installed in a secure computer network. The system administrator must have elevated system privileges.
 
-* The YubiHSM 2 SDK  downloaded from the Yubico YubiHSM 2 Release page and available on the system to be used. Installation instructions are given in the following.
+* The YubiHSM 2 SDK  downloaded from https://developers.yubico.com/YubiHSM2/Releases/ [the Yubico YubiHSM 2 Release page] and available on the system to be used. Installation instructions are given in the following.
```

Comment:
This doesn't render as a namned link properly, maybe we could make it relative as well?

Reply:
Fixed: link:../../Releases/[the Yubico YubiHSM 2 Release page]